contracts/cw20-base/src/enumerable.rs

```diff
@@ -143,6 +172,71 @@ mod tests {
         assert_eq!(&allow.allowance, &allow2);
     }
 
+    #[test]
+    fn query_all_spender_allowances_works() {
+        let mut deps = mock_dependencies_with_balance(&coins(2, "token"));
+
+        // these are in alphabetical order same than insert order
+        let owner1 = String::from("owner1");
+        let owner2 = String::from("owner2");
+        let spender = String::from("spender");
+
+        let info = mock_info(owner1.as_ref(), &[]);
+        let env = mock_env();
+        do_instantiate(deps.as_mut(), &owner1, Uint128::new(12340000));
+
+        // no allowance to start
+        let allowances =
+            query_spender_allowances(deps.as_ref(), spender.clone(), None, None).unwrap();
```

Comment:
You'd get rid of those codecov warnings if you'd pass QueryMsg instead of calling method directly.

Reply:
Only because you asked for it.